docs/howto/features/buckets.md

```diff
@@ -32,7 +32,16 @@ on why users want this!
    in the same `.tfvars` file. Follow all the steps listed there - this
    should create the storage buckets and provide all users access to them!
 
-3. You can set the `SCRATCH_BUCKET` (and the deprecated `PANGEO_SCRATCH`)
+3. (If requested) Enable public read access to these buckets by editing the
+   `bucket_public_access` list in the same `.tfvars`:
+
+   ```terraform
+   bucket_public_access = [
+      "persistent-public"
```

Comment:
To match `PUBLIC_BUCKET: gs://m2lines-public/$(JUPYTERHUB_USER)`, this would need to be like this I think:

```suggestion
      "public"
```

I think I'd favor the more brief `public` overall.

Reply:
@consideRatio, if we were to have public buckets that aren't persistent, woudn't be better to be explicit about this?
My motivation is that there can be buckets that are:
- scratch
- persistent 
- public + scratch
- public + persistent

I'm now thinking if it shouldn't in fact be the other way around, i.e. have the env var match this composed pattern, i.e. a PUBLIC_PERSISTENT_BUCKET and a PUBLIC_SCRATCH_BUCKET? 😅

Reply:
I agree with Georgiana. Better to be explicit now than have someone complain that their data was wiped, or made public when it shouldn't, because of unclear naming conventions leading to using the inappropriate bucket. Even if we later find out it's not needed, at least we won't find out we made the wrong choice because someone's work was ruined.

(I also agree with the switching the word order, it syntactically reads better :) )

Reply:
@GeorgianaElena I've updated my comment after you replied to it!

I like the consistency you suggest, but i guess just like how we omit `private` or `local` etc for the non-public buckets, I was thinking we may want to not declare `persistent` because its kind of what you would assume - and could be misleading for someone in the public that may think that data URLs are immutable and that is the meaning of persistent.

Reply:
Okay seeing @sgibson91's input I think we should not think further and go with what you outlined above @GeorgianaElena!

Thanks for helping out discussing thing @sgibson91!!!

I think we should use an env var name that is directly associated with the name of the bucket, so lets make the env var `PUBLIC_PERSISTENT_BUCKET` then as well - right?